elm-format-lib/src/Parse/Number.hs

```diff
@@ -129,7 +129,7 @@ chompInt !pos end !n =
       if isDecimalDigit word then
         chompInt (plusPtr pos 1) end (10 * n + fromIntegral (word - 0x30 {-0-}))
 
-      else if word == 0x2E {-.-} then
+      else if word == 0x2E {-.-} && not (isDot (plusPtr pos 1) end) then
```

Comment:
👍 And I think this is safe because previously going into `chompFraction` with a second `.` would always result in an error, and the only change now is that if there is a valid range syntax, that will be parsed as valid?

Reply:
Yes, that is correct.

This behavior of looking ahead for a second `.` also retains the same behavior as [Parse.Literals.decimals](https://github.com/avh4/elm-format/pull/747/commits/e691d73cda22d9a05cb3b53f7e28c00cfb9c5ab2#diff-aaaf99696880972a84fa197f3fb95b771e2ee24e7fbc6cc5e59ffe520c92c835L64) which was the function that parsed the decimal bit previously